<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TWLTheme</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="iconViewBattery">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>20</y>
      <width>18</width>
      <height>11</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>18</width>
      <height>11</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>18</width>
      <height>11</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>261</width>
      <height>540</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>18</width>
      <height>0</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QGraphicsView" name="bgViewBottom">
       <property name="minimumSize">
        <size>
         <width>18</width>
         <height>192</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>192</height>
        </size>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGraphicsView" name="bgViewTop">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>18</width>
         <height>192</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>192</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QPushButton" name="bgButtonBottom">
         <property name="minimumSize">
          <size>
           <width>18</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Bottom Background</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="saveButton">
         <property name="minimumSize">
          <size>
           <width>18</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="bgButtonTop">
         <property name="minimumSize">
          <size>
           <width>18</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Top Background</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="iconButtonBattery">
         <property name="minimumSize">
          <size>
           <width>18</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Battery Icon</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>gridLayoutWidget</zorder>
   <zorder>iconViewBattery</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
